theSrc/scripts/Pictograph.js

```diff
@@ -76,7 +77,7 @@ class Pictograph {
       .then(this._computeCellPlacement.bind(this))
       .then(this._render.bind(this))
       .catch((error) => {
-        if (error.type === InsufficientContainerSizeError.type) {
+        if (error.type === InsufficientContainerSizeError.type || error.type === UserError.type) {
           console.log(error.message)
           d3.select(this.rootElement).attr(`rhtmlwidget-status`, 'ready')
           d3.select(this.rootElement).attr(`rhtmlPictographs-status`, 'ready') // to be removed once regression testing code checks for rhtmlwidget-status
```

Comment:
Can we remove this line since I think it is no longer needed?

Reply:
Yep, removed rhtmlPictographs-status now